### PR TITLE
clarifications on TLS usage, remove Status List Endpoint

### LINKED ISSUE #120: Status List Endpoint is undefined
Section [8.1. ](https://www.ietf.org/archive/id/draft-ietf-oauth-status-list-02.html#section-8.1)[Status List Request](https://www.ietf.org/archive/id/draft-ietf-oauth-status-list-02.html#name-status-list-request) mentions Status List Endpoint. The term is not mentioned elsewhere in the specification nor is it defined.

### LINKED ISSUE #126: Remove the requirement to use TLS to retrieve Status List Tokens from the Status List Endpoint
Section '8.1. Status List Request' currently states:

> Communication with the Status List Endpoint MUST utilize TLS.

This seems redundant when the endpoint is used to retrieve a Status List Token, as the Token's integrity is already guaranteed by requiring it to be signed using an asymmetric cryptographic algorithm.

If requiring TLS was intended for providing confidentiality of the Token, this seems unnecessary. The endpoint is publicly accessible - there does not seem to be any client authentication mechanisms described for it - thus the Tokens could be retrieved by anybody.

### PR DESCRIPTION
**Pull request by paulbastian**
"endpoint" is still mentioned in there, unsure we we completely want to get rid of it

Closes #120 
Closes #126

### CODE REVIEW

File: draft-ietf-oauth-status-list.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ The status list token data structures themselves are also represented as JWTs or
...
+This document defines a Status List and its representations in JSON and CBOR formats that describe the individual statuses of multiple Referenced Tokens, which themselves are JWTs or CWTs. The statuses of all Referenced Tokens are conveyed via a bit array in the Status List. Each Referenced Token is allocated an index during issuance that represents its position within this bit array. The value of the bit(s) at this index correspond to the Referenced Token's status. A Status List may either be provided by an HTTPS endpoint or be signed and embedded into a Status List Token, whereas this document defines its representations in JWT and CWT. Status Lists may be composed for expressing a range of Status Types. This document defines basic Status Types for the most common use cases as well as an extensibility mechanism for custom Status Types. The document also defines how an issuer of a Referenced Token references a Status List (Token).
```
Comment (c2bo) [inline draft-ietf-oauth-status-list.md:85]:
```suggestion
This document defines a Status List and its representations in JSON and CBOR formats that describe the individual statuses of multiple Referenced Tokens, which themselves are JWTs or CWTs. The statuses of all Referenced Tokens are conveyed via a bit array in the Status List. Each Referenced Token is allocated an index during issuance that represents its position within this bit array. The value of the bit(s) at this index correspond to the Referenced Token's status. A Status List may either be provided via HTTPS or be signed and embedded into a Status List Token, whereas this document defines its representations in JWT and CWT. Status Lists may be composed for expressing a range of Status Types. This document defines basic Status Types for the most common use cases as well as an extensibility mechanism for custom Status Types. The document also defines how an issuer of a Referenced Token references a Status List (Token).
```

Comment (c2bo) [reply draft-ietf-oauth-status-list.md:85]: not sure about "HTTPS endpoint"

File: draft-ietf-oauth-status-list.md [hunk 2]
```diff
@@ -427,7 +427,9 @@ The processing rules for JWT or CWT precede any evaluation of a Referenced Token
 
 ## Status List Request
 
-To obtain the Status List or Status List Token, the Relying Party MUST send a HTTP GET request to the Status List Endpoint. Communication with the Status List Endpoint MUST utilize TLS. Which version(s) should be implemented will vary over time. A TLS server certificate check MUST be performed as defined in Section 5 and 6 of {{RFC6125}}.
+To obtain the Status List or Status List Token, the Relying Party MUST send a HTTP GET request to the URI provided in the Referenced Token.
```
Comment (c2bo) [inline draft-ietf-oauth-status-list.md:430]:
```suggestion
To obtain the Status List or Status List Token, the Relying Party MUST send an HTTP GET request to the URI provided in the Referenced Token.
```

File: draft-ietf-oauth-status-list.md [hunk 3]
```diff
@@ -795,6 +797,11 @@ for their valuable contributions, discussions and feedback to this specification
 # Document History
 {:numbered="false"}
 
+-03
+
+* require TLS for fetching only for Status List, not for Status List Token
```
Comment (c2bo) [inline draft-ietf-oauth-status-list.md:802]:
```suggestion
* require TLS only for fetching Status List, not for Status List Token
```

**Review by c2bo — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Pull request merged**